core/include/usBundle.h

```diff
@@ -192,36 +347,33 @@ class US_Core_EXPORT Bundle : public std::enable_shared_from_this<Bundle>
   /**
    * Returns this bundle's location.
    *
-   * <p>
    * The location is the full path to the bundle's shared library.
    * This method continues to return this bundle's location
-   * while this bundle is in the <code>STOPPED</code> state.
+   * while this bundle is in the <code>STATE_STOPPED</code> state.
    *
    * @return The string representation of this bundle's location.
    */
   virtual std::string GetLocation() const;
 
   /**
-   * Returns the name of this bundle as specified by the
-   * US_CREATE_BUNDLE CMake macro. The bundle
+   * Returns the symbolic name of this bundle as specified by the
+   * US_BUNDLE_NAME preprocessor definition. The bundle symbolic
    * name together with a version must identify a unique bundle.
    *
-   * <p>
-   * This method continues to return this bundle's name while
-   * this bundle is in the <code>STOPPED</code> state.
+   * This method continues to return this bundle's symbolic name while
+   * this bundle is in the <code>STATE_STOPPED</code> state.
    *
-   * @return The name of this bundle.
+   * @return The symbolic name of this bundle.
    */
-  std::string GetName() const;
+  std::string GetSymbolicName() const;
 
   /**
-   * Returns the version of this bundle as specified by the
-   * US_INITIALIZE_BUNDLE CMake macro. If this bundle does not have a
+   * Returns the version of this bundle as specified in its
+   * <code>manifest.json</code> file. If this bundle does not have a
    * specified version then {@link BundleVersion::EmptyVersion} is returned.
    *
-   * <p>
    * This method continues to return this bundle's version while
-   * this bundle is in the <code>STOPPED</code> state.
+   * this bundle is in the <code>STATE_STOPPED</code> state.
```

Comment:
What about when the bundle is in STATE_UNINSTALLED ?? 

Here's the OSGi javadoc comment for this method ....

"Returns the version of this bundle as specified by its Bundle-Version manifest header. If this bundle does not have a specified version then Version.emptyVersion is returned.
This method must continue to return this bundle's version while this bundle is in the UNINSTALLED state."